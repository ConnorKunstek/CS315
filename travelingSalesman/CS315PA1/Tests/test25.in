Archidiaconate 36.3782 -84.0296
Prophetic 36.4016 -75.3438
Waylay 33.7106 -76.6118
Demiram 32.9735 -88.2774
Canonry 38.3321 -89.0699
Landuman 32.7395 -78.07
Unfinishedness 36.4484 -73.315
Hornlike 33.4181 -87.0411
Tret 34.6115 -74.0758
Superstatesman 32.9618 -87.2313
Cerolite 35.9453 -87.7385
Catalogue 33.9914 -75.9461
Hulk 36.7175 -79.3063
Suspicional 37.0919 -86.8509
Molybdomenite 37.9343 -78.6406
Cod 36.1442 -77.0556
Quinology 35.9219 -80.2256
Greenly 34.9742 -79.8135
Neckatee 36.3197 -73.9173
Lurchline 37.3025 -88.1823
Intropulsive 34.4477 -73.5369
Crambambuli 38.3906 -80.0354
Undiminishable 32.7746 -75.0902
Perissodactylous 39.0809 -87.6751
Worble 33.7925 -89.1016
