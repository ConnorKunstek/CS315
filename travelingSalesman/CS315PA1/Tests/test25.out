

West to East: Worble to Unfinishedness
Distance in Kilometers: 7686 km
Distance in Miles: 4776 miles
North to South: Perissodactylous to Landuman
Distance in Kilometers: 14207 km
Distance in Miles: 8828 miles
Time: 0.000577 seconds

